### PR TITLE
fix: Fixes break from JSR-9

### PR DESCRIPTION
**Pull request by mathiuskitchens**
- Fixes breaking error caused by my poor de-structuring
- Removed `container` items from `tailwindconfig.js`

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -72,6 +72,8 @@
         ],
         "react/no-unescaped-entities": "off",
         "react/require-default-props": "error",
+        "react/jsx-uses-react": "error",
```
Comment (wijohnst) [inline .eslintrc.json:75]: I'm not really sure what this one does. I read the docs and I'm still confused.

File: package-lock.json [hunk 2]
```diff
@@ -5029,6 +5029,18 @@
         "url": "https://github.com/sponsors/ljharb"
       }
     },
+    "node_modules/array.prototype.tosorted": {
```
Comment (wijohnst) [inline package-lock.json:5032]: Always try to include any dependencies you add in your PR description. 

What did you add? Why did you add it? Stuff like that.

**Review by wijohnst — COMMENTED** [review]
Only thing I'm confused about is the strange dependency.

File: .eslintrc.json (hunk 1 above)
Comment (mathiuskitchens) [reply .eslintrc.json:75]: I think this one I tried to attempt to resolve my error with props.children and forgot to remove it

File: package-lock.json (hunk 2 above)
Comment (mathiuskitchens) [reply package-lock.json:5032]: Oh I think I did add one. This was also to try and fix my silly error. It should be this one: https://www.npmjs.com/package/eslint-plugin-react

**Review by wijohnst — APPROVED** [review]
LGTM!
